### ISSUE TITLE
Add [darktable]/[Design Tools]

### ISSUE THREAD

**Issue by Helenaria** [pluja/awesome-privacy#371]
### DarkTable

* [darktable](https://www.darktable.org/) - darktable is an open source photography workflow application and raw developer. A virtual lighttable and darkroom for photographers. It manages your digital negatives in a database, lets you view them through a zoomable lighttable and enables you to develop raw images and enhance them.
* **License**: <GPL 3.0>
* **Why do you think this helps users privacy?**: <Open source, commonly used alongside RawTherapee as Adobe Lightroom replacements>
* **Under what section should this service be listed?**: <Design Tools>
* **Additional comments / info**: Github page [here](https://github.com/darktable-org/darktable).

![image](https://github.com/pluja/awesome-privacy/assets/80150720/365b7e1c-a723-4e0c-86d8-892493931d3c)

**Comment by pluja** [comment]
Just as rawtherapee, wouldn't [Photo Editing and Management](https://github.com/pluja/awesome-privacy#photo-editing-and-management) be a better fit?

**Comment by Helenaria** [comment]
Yeah, that's right! I wrote both as design tools given i first saw them recommended for designing. Feel free to place them wherever they fit best.
